ios/brave-ios/Sources/Brave/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -96,7 +96,7 @@ class InitialSearchEngines {
   }
 
   let braveSearchDefaultRegions = [
-    "US", "CA", "GB", "FR", "DE", "AD", "AT", "ES", "MX", "BR", "AR", "IN", "IT",
+    "US", "CA", "GB", "FR", "DE", "AU", "AT", "ES", "MX", "BR", "AR", "IN", "IT",
```

Comment:
Chromium Brave override specifies Brave search as a default for `AU` here: https://github.com/brave/brave-core/blob/master/chromium_src/components/regional_capabilities/regional_capabilities_utils.cc#L650. 
Also Chromium Brave override specifies nothing for `AD`.